logline.go

```diff
@@ -192,6 +191,7 @@ func (l *LogLine) ToText(buf []byte) []byte {
 	return buf
 }
 
+// Clear the logline, overwriting with a new logline, only inheriting the field separator
```

Comment:
```suggestion
// Clear clears the logline...
```

The fact that it's overwritten with a new logline is an implementation detail.

FYI: On `Record.Clear` the comment is 
> // Clear clears the record internal state, making it empty.